jupyterhub/spawner.py

```diff
@@ -174,12 +263,16 @@ def _deprecate_percent_u(self, proposal):
             v = v.replace('%U', '{username}')
             self.log.warning("Converting %r to %r", proposal['value'], v)
         return v
-    
-    disable_user_config = Bool(False,
-        help="""Disable per-user configuration of single-user servers.
-        
-        This prevents any config in users' $HOME directories
-        from having an effect on their server.
+
+    disable_user_config = Bool(
+        False,
+        help="""
+        Disable per-user configuration of single-user servers.
+
+        This prevents any config in users' $HOME directories from having an effect on their server.
```

Comment:
In general, if something looks like it could be used for anything security related, I always try to put a note explaining the cases where it would *not* work right there.